src/core/worker.js

```diff
@@ -106,6 +106,22 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
 
           var length = fullRequestXhr.getResponseHeader('Content-Length');
           length = parseInt(length, 10);
+//#if !(FIREFOX || MOZCENTRAL)
```

Comment:
NIt: since the code shouldn't be necessary in FirefoxOS either, this line should *ideally* be: `//#if !(FIREFOX || MOZCENTRAL || B2G)`.